### ISSUE TITLE
Tabbing to properties with no value cannot see the keyboard focus on properties dialog of blob snapshots 

### ISSUE THREAD

**Issue by v-xuanzh** [microsoft/AzureStorageExplorer#1284]
**Storage Explorer Version:** 1.7.0_201904010.5
**Platform/OS:** Linux Ubuntu/macOS High Sierra/Windows 10
**Architecture:** ia32/x64
**Regression From:** Not a regression

**Steps to reproduce:**
1. Change the current theme to HC-Black/White.
2. Open one normal blob container -> Create a snapshot for one blob.
3. Open the properties dialog of the created snapshot  -> Press Tab to check the properties.

**Expect Experience:**
We can see the keyboard focus.

**Actual Experience:**
We cannot see the keyboard focus when tabbing to the properties with no value.
![image](https://user-images.githubusercontent.com/34729022/55943850-3c1e0f80-5bfc-11e9-8390-4bfe8d799258.png)

**More info:**
This issue doesn't reproduce on Light/Dark theme.

**Comment by v-xuanzh** [comment]
This issue has been fixed on the latest master build 20190723.4. So closed it.

**Comment by v-limu** [comment]
This issue has been fixed on build [20191128.3](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3280503&view=results). So we close it.
